docs/collections/contribute.md

```diff
@@ -1,14 +1,116 @@
 
-# Contributing Collections
+# Contribute
+
```

Comment:
I'd like to see some introduction text between headings, even if just a sentence. "You can contribute blocks and collections by following these steps." Or words to that effect is fine.